src/components/Recover.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+import {NavLink} from "react-router-dom";
+import axios from 'axios';
+import './recover.scss';
+import {sentResetLinkText} from "./utilities-texts";
+
+const {REACT_APP_RESET_PASS_PATH} = process.env;
+
+
+class Recover extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            email: '',
+            isMessageShown: false,
+            message: '',
+            messageColor: ''
+        };
+
+        this.change = this.change.bind(this);
+        this.submit = this.submit.bind(this);
+    }
+
+    change(e) {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    showMessage = (msg) => {
+        this.setState({
+            isMessageShown: true,
+            message: msg
+        })
+    }
+
+    setMessageColor = (color) => {
+        this.setState({messageColor: color})
+    }
+
+    showSuccessMessage = () => {
+        this.setState({isMessageShown: true});
+    };
+
+    hideMessage = () => {
+        if (this.state.isMessageShown) {
+            this.setState({
+                isMessageShown: false,
+                message: ''
+            })
+        }
+    }
+
+    submit(e) {
+        e.preventDefault();
+        const url = REACT_APP_RESET_PASS_PATH;
+
+        axios.post(url,
+            {
+                "Email": this.state.email,
+            },
+            {
+                headers: {
+                    'Cache-Control': 'no-cache',
+                    'Content-Type': 'application/json',
```

Comment:
This header is always present in POST request. I mean that it should be used in some kind of generic method. It's not a high priority.